packages/manager/apps/pci-rancher/src/routes.tsx

```diff
@@ -112,6 +112,12 @@ export default [
           },
         ],
       },
+      {
+        path: 'rancher/:rancherId/update-software',
+        ...lazyRouteConfig(() =>
+          import('@/pages/dashboard/updateSoftware/UpdateSoftware.page'),
+        ),
+      },
```

Comment:
**nitpick:** According to the folder structure, "Update Software" looks like a child of Dashboard. Can you make it consistent with the folder structure?